wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -412,10 +415,10 @@ public final class ZMUserSession: NSObject {
         ) { [weak self] in
             guard let context = self?.syncContext else { return }
 
-            context.perform {
-                let service = SupportedProtocolsService(context: context)
-                service.updateSupportedProtocols()
-            }
+//            context.perform {
+//                let service = SupportedProtocolsService(context: context)
+//                service.updateSupportedProtocols()
+//            }
```

Comment:
private func updateSupportedProtocolsIfNeeded() needs to be deleted entirely @johnxnguyen

Reply:
I just did it!